modules/apps/object/object-test/src/testFunctional/tests/ObjectPermission.testcase

```diff
@@ -1,4 +1,6 @@
 @component-name = "portal-object"
+definition {
+@component-name = "portal-object"
```

Comment:
This is what's causing your SF failure. You have duplicated lines (1-2 and 3-4) and therefore an unclosed boundary "{}". Please fix.

Reply:
Oh man, thanks for this. I really don't noticed this 😅